test-scripts/tomcat-deployment/README.md

```diff
@@ -0,0 +1,19 @@
+Dummy Scripts to Test TG execution
```

Comment:
Shall we make this a title (## Dummy Scripts ...)?

Reply:
This is Fixed with 90a076769115d759559419fa046eaf72e213abbb